modules/ROOT/pages/test-recorder.adoc

```diff
@@ -0,0 +1,15 @@
+= Test Recorder in Studio (Beta)
```

Comment:
@kip-mulesoft This is an idea of the landing page for the Test recorder.
It might be a good fit for the "Learn More" link in the wizard. 

I'll add more information about the test recorder itself. Do you think it would be good to list the Limitations of the tool being beta?

Reply:
I think this is the best place to put limitations, especially if this is where we point our learn more link.